### ISSUE TITLE
Automated install bug: fails with "file" argument must be of type string

### ISSUE THREAD

**Issue by SynoUser-NL** [saltcorn/saltcorn#1877]
**Describe the bug**
Tried installing Saltcorn with automated install on Debian11. 

**To Reproduce**
0. Install fresh Debian11 on a new VM
1. Go to ['...'](https://wiki.saltcorn.com/view/ShowPage/install-on-ubuntu)
2. Follow Quick automated install, on step 3: npx saltcorn-install -y, it fails with the error:
The "file" argument must be of type string. Received null

**Expected behavior**
To have a working saltcorn site..

**Additional context**
The same error is generated when doing npx saltcorn-install, but now the script fails after setting the deployment option as Server Mode and setting the Web port on 80.
The line apt install -qqy ... {etc}.. is generated, after which the error is displayed..

The same happens after manually installing all the apt install components mentioned and restarting the npx command.

**Comment by serge402** [comment]
@SynoUser-NL Any particular reason for using Debian11?

If you are OK with using Ubuntu you can follow my instructions here #1830
I have Saltcorn working with all functions including the Mobile App builder.

**Comment by SynoUser-NL** [comment]
Yes.. because Ubuntu does not install well on my Synology as a VM (it is slow as ...) and Debian9-10-11 do (blazingly fast). I use Debian in most cases and prefer it greatly over Ubuntu.

Besides, the online installation manual claims the installation has been tested on Debian 11.. which I now can only surmise it has not been. And I am not even installing the Mobile App builder.

I also tried the docker install which also does not work.

I'd really like to try this software, however all the above doesn't inspire much confidence in the product even before installing it.

**Comment by serge402** [comment]
@SynoUser-NL That's odd, I use a Synology NAS to run my Ubuntu VM to host Saltcorn, which seems to run fine.
I haven't tried to install Debian vs Ubuntu on my Synology, so can't comment on speed. Maybe it's a setting for VM?

Yeah, I haven't managed to get Docker working ether.

If you can put up with Ubuntu install or try it in a VM on a local machine, otherwise Saltcorn works great. (minor bugs included)

**Comment by glutamate** [comment]
@SynoUser-NL I can't replicate this. We build in a bunch of vagrant boxes and report the results here: https://releases.saltcorn.com/. For the latest, https://releases.saltcorn.com/view/ListBuildResults?release=6 you see debian 11 passing (see "show output") - maybe see if what you did was different than in the Vagrantfile script?

**Comment by SynoUser-NL** [comment]
Hi,
Thanks for your comment. I've since switched to Ubuntu Server 20.04 on which I got Saltcorn to work, with a bit of fiddling and help from the post of serge402. 

Most obvious change from the online install doc is the switch to Node 16 (from 14..).
My Debian11 was a fresh install from downloaded ISO, and updated components.

I'll close this issue with the comment that Ubuntu Server works fine. :)

**Comment by danpertech** [comment]
Has the cause of this problem been identified? I've installed Saltcorn several times on Debian and Ubuntu without any issues. But recently I tried installing Saltcorn on Proxmox, either in a container (CT) or virtual machine (VM), and I keep getting the same error, both with Debian and Ubuntu.

root@saltcorn:~# npx saltcorn-install -y
> apt install -qqy libpq-dev build-essential python-is-python3 git libsystemd-dev zip unzip chromium libcap2-bin postgresql postgresql-client
The "file" argument must be of type string. Received null
